src/System.Drawing.Common/pkg/System.Drawing.Common.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Drawing.Common.csproj">
-      <SupportedFramework>netcoreapp2.0</SupportedFramework>
+      <SupportedFramework>net461;netcoreapp2.0</SupportedFramework>
```

Comment:
@weshaggard should I add any other supported frameworks? Like AllXamarinFrameworks?

Reply:
I don't believe so. We generally only add those when the Xamarin folks ask us to because they put this assembly inbox. @marek-safar @akoeplinger can correct me if I'm wrong.

Reply:
@weshaggard is this a new netstandard assembly/package?

Reply:
Yes it is a new package that targets netstandard. We have a few new packages that we should provide you a list of so you know what is coming.

Reply:
I've talked to @marek-safar offline and I shared a list of what packages that target netstandard are coming.

Reply:
@weshaggard @marek-safar I've added `AllXamarinFrameworks` to the pkgproj and added them as inbox in the package index as well. Please let me know if looks good.